ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -44,7 +44,7 @@
   private AdConfig mAdConfig;
   private String mPlacementForPlay;
 
-  private VungleBannerAdapter vungleBannerAdapter;
+  private VungleBannerAdapterProxy vungleBannerAdapter;
```

Comment:
we could also extend VungleInterstitialAdapter from VungleBannerAdapterProxy(current name) to get rid of this variable and easier separation in future

Reply:
and remove all the     if (vungleBannerAdapter != null) { } checks

Reply:
deleted VungleBannerAdapterProxy file and have delegated all banner requests to VungleBannerAdapter class.